app/Http/Controllers/FileController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class FileController extends Controller
+{
+    public function upload(Request $request)
+    {
+        if (empty($request->file('Invoice'))){
+
+            return response()->json(['message' => 'Keine Datei zum Server Geschickt'], 404);
```

Comment:
use english in code, even for messages: they will be translated in frontend if necessary